samples/v1.2/Elements/Input.Text.InlineAction.json

```diff
@@ -36,7 +36,7 @@
 		},
 		{
 			"type": "Input.Text",
-			"id": "input1",
+			"id": "input3",
```

Comment:
have you rerun the UWP tests to make sure this is reflected properly?

Reply:
will do it; thanks!

---
In reply to: [229816886](https://github.com/Microsoft/AdaptiveCards/pull/2136#discussion_r229816886) [](ancestors = 229816886)